test/generators/shop_model_generator_test.rb

```diff
@@ -29,6 +33,13 @@ class ShopModelGeneratorTest < Rails::Generators::TestCase
     end
   end
 
+  test "injects into application controller" do
+    run_generator
+    assert_file "app/controllers/application_controller.rb" do |controller|
+      assert_match "def shop", controller
+    end
```

Comment:
v6 shipped in April 2015 which was the first version of the more modern rails engine. This major version also modularized the generators and included a shop modal generator which was ran by default. So effectively unless someone on purpose removed the shop modal anyone who generated an app in the last year would have the standard shop model.

Does that answer you question? It would be simpler on this code base to force the shop model and it is something I have considered before.